cob_default_robot_config/cob4-1/head_joint_configurations.yaml

```diff
@@ -1 +0,0 @@
-../cob4-2/head_joint_configurations.yaml
```

Comment:
as the HW for head is the same for cob4-1 and cob4-2, we sould use softlinks (cob4-2 is original and cob4-1 is linked to cob4-2)